org.telegram.desktop.yml

```diff
@@ -346,26 +326,34 @@ modules:
           versions:
             <: 2.73.0
 
-  - name: mozjpeg
-    buildsystem: cmake-ninja
-    builddir: true
-    config-opts:
-      - -DCMAKE_BUILD_TYPE=None
-      - -DCMAKE_INSTALL_DEFAULT_LIBDIR=lib
-      - -DENABLE_STATIC:BOOL=OFF
+  - name: jemalloc
+    buildsystem: autotools
     sources:
       - type: archive
-        url: https://github.com/mozilla/mozjpeg/archive/v4.1.1/mozjpeg-4.1.1.tar.gz
-        sha256: 66b1b8d6b55d263f35f27f55acaaa3234df2a401232de99b6d099e2bb0a9d196
+        url: https://github.com/jemalloc/jemalloc/releases/download/5.3.0/jemalloc-5.3.0.tar.bz2
+        sha256: 2db82d1e7119df3e71b7640219b6dfe84789bc0537983c3b7ac4f7189aecfeaa
         x-checker-data:
-          type: json
-          url: https://api.github.com/repos/mozilla/mozjpeg/releases/latest
-          tag-query: .tag_name
-          timestamp-query: .published_at
-          version-query: $tag | sub("^[vV]"; "")
-          url-query: '"https://github.com/mozilla/mozjpeg/archive/\($tag)/mozjpeg-\($version).tar.gz"'
+          type: anitya
+          project-id: 1441
+          url-template: https://github.com/jemalloc/jemalloc/releases/download/$version/jemalloc-$version.tar.bz2
     cleanup:
       - /bin
+      - /share
+
+  - name: openssl11
```

Comment:
I wonder how that affects the webview addon, wouldn't it happen that webkitgtk builds with openssl3 while tdesktop itself builds with openssl 1.1 and everything crashes once the user opens a web bot?

Reply:
Not sure what will happen, but this is how is packed in Arch Linux, but not sure if it fails in those cases you mention...
We will need to test the build, I will upload the other changes, and keep this one.

Even that the flatpak builds successfully without this module, I believe this is required.

Reply:
I mean we have an extension: https://github.com/flathub/org.telegram.desktop.webview
I thought it uses openssl as well, but it seems it uses gnutls actually...